views/get-started.erb

```diff
@@ -25,35 +25,65 @@
 		<div class="column-two-thirds">
 			<h1 class="heading-xlarge">Get started with GOV.UK PaaS</h1>
 			<h2 class="heading-large">Evaluate GOV.UK PaaS</h2>
+			<h3 class="heading-medium">Check our features and pricing</h3>
 			<p>
-				If you’re considering cloud-based solutions for your web app, GOV.UK PaaS offers a trial period with a set of basic features and a capped <a href="https://docs.cloud.service.gov.uk/managing_apps.html#quotas">quota</a> of compute and storage. Keep in mind that you can’t use the trial version of the platform for production or live services.
+				Make sure we currently support the <a href="https://docs.cloud.service.gov.uk/before_you_start.html#before-you-start">language, framework</a> and <a href="/features">backing services</a> you need. Our <a href="/roadmap">roadmap</a> lists the features planned for release in the coming months.
 			</p>
-
-			<h3 class="heading-medium">Check the features</h3>
+			
 			<p>
-				Make sure we currently support the <a href="https://docs.cloud.service.gov.uk/before_you_start.html#before-you-start">language, framework</a> and <a href="/features">backing services</a> you need. Our <a href="/roadmap">roadmap</a> lists the features planned for release in the coming months.
+				You can also find out <a href="/pricing">how our pricing works</a> and <a href="https://admin.london.cloud.service.gov.uk/calculator">estimate your monthly costs</a>.
+			</p>	
+			
+			<p>
+				If you need more information to support your evaluation, please <a href="/support/find-out-more">get in contact with us</a>.
 			</p>
 
-			<h3 class="heading-medium">Request an account</h3>
+			<h3 class="heading-medium">Request a free trial account</h3>
 			<p>
-				When you <a href="/signup">request an account</a> you can invite as many team members as you need and choose the ones who’ll be in charge of managing users. They’ll be able to request more user accounts, including ones for external suppliers, and assign <a href="https://docs.cloud.service.gov.uk/orgs_spaces_users.html#users-and-user-roles">roles and permissions</a>.
+				GOV.UK PaaS offers a 3 month trial period with a set of basic features and a capped quota of compute and storage.
 			</p>
-
+			
+			<p>
+				During this period you can test the majority of the features, including:
+			</p>
+			
+			<ul class="list list-bullet">
+			<li>managing user accounts</li>
+			<li>creating and managing environments</li>
+			<li>deploying and running apps</li>
+			<li>using trial-sized editions of our backing services</li>
```

Comment:
I wonder about flipping the order of these around: deploying/running apps and using backing services is what appeals to people new to PaaS; user management is more of a concern once you know PaaS is ❤️.